### DISCUSSION TITLE
Taglist

### DISCUSSION THREAD

**Discussion by austyuzhaninov** [moonshine-software/moonshine#204] [Ideas]
День добрый, подскажите существует ли в проекте taglist?

Установил пакеты:
spatie/laravel-tags: 4.3.7
spatie/laravel-translatable: 6.4.0
visual-ideas/moonshine-spatie-translatable: 1.0.0

В Laravel создал модель Tag, для неё ресурс в админке. Реализовал через отношение через BelongsToMany:
![image](https://user-images.githubusercontent.com/38152348/233183529-cf3b72ae-b084-4f6b-9330-c03969878d25.png)

В текущей реализации можно только выбрать тег, как понимаю создавать нельзя. Пройдясь по доке не нашел похожего свойства. Было бы круто иметь что-то на подобия taglist как в OctoberCMS, чтобы в одной форме создавать теги.
![image](https://user-images.githubusercontent.com/38152348/233183918-8f82954c-58d6-404e-8f3a-c7b407876b79.png)

**Comment by lee-to** [comment]
Хорошая идея, добавим в ближайшие задачи

**Comment by Sanes** [comment]
Когда выбирал пакет для тегов, `cviebrock/eloquent-taggable` показался удобней, чем `spatie/laravel-tags`

**Comment by lee-to** [comment]
https://github.com/lee-to/moonshine-spatie-tags-field

**Reply by ermakk** [reply]
Будет ли реализация для MoonShine 2?

**Reply by lee-to** [reply]
Она не требуется, есть метод tags у поля Text и через методы changeFill, onApply можно легко самостоятельно интегрировать spatie-tags
Не хочется делать пакет из 10 строчек кода)

**Reply by ermakk** [reply]
Спасибо) Не заметил что у Text есть метод tags)
